pkg/plugins/builtin/ssh/README.md

```diff
@@ -2,6 +2,8 @@
 
 This plugin connects to a remote system and performs a block of commands. It can extract variables from the shell back to the output of its enclosing step.
 
+Task will be considered successful if script returns exit code 0, otherwise, it will be considered as SERVER_ERROR. For some unrecoverable errors (for instance, invalid parameters), it is possible to add a configuration (`exit_codes_unrecoverable`) to consider some exit codes as CLIENT_ERROR.
```

Comment:
```suggestion
The step will be considered successful if the script returns exit code 0, otherwise, it will be considered as a `SERVER_ERROR` (and will be retried). For unrecoverable errors (for instance, invalid parameters), it is possible to configure a list of exit codes (see `exit_codes_unrecoverable`) that should halt the execution (`CLIENT_ERROR`).
```